Mathlib/Data/Finset/Basic.lean

```diff
@@ -3850,6 +3859,36 @@ def sigmaEquivOptionOfInhabited (α : Type u) [Inhabited α] [DecidableEq α] :
           · simp }⟩
 #align equiv.sigma_equiv_option_of_inhabited Equiv.sigmaEquivOptionOfInhabited
 
+variable [DecidableEq α] {s t : Finset α}
+
+/-- `s ∪ t` (using finset union) is equivalent to `s ∪ t` (using set union) -/
+@[simps!]
+def finsetUnion (s t : Finset α) :
+    (s ∪ t : Set α) ≃ ((s ∪ t : Finset α) : Set α) :=
+  Equiv.Set.ofEq <| coe_union _ _ |>.symm
```

Comment:
I'm not convinced this one is worth having; the space of equivs which can be proved with `Equiv.Set.ofEq` seems too large to justify giving any of them names.